project1/project1/application.py

```diff
@@ -34,16 +42,18 @@ def register():
         username = request.form["username"]
         password = request.form["password"]
         timeStamp = datetime.datetime.now()
-        form = Form(username=username, password=password, timeStamp=timeStamp)
+        obj = User(username=username, password=password, timeStamp=timeStamp)
         str = username + " " + "entered details"
 
         try:
-            session.add(form)
-            session.commit()
+            db.session.add(obj)
+            db.session.commit()
             # print("committed")
             return render_template("success.html", message = str)
         except:
             return render_template("error.html")
     else:
         return render_template("registration.html")
```

Comment:
Show the messages in the registration page itself dont use other html pages to display the messages